tsdb/engine/tsm1/iterator.gen.go

```diff
@@ -223,6 +224,17 @@ func newFloatIterator(name string, tags query.Tags, opt query.IteratorOptions, c
 	itr.conds.names = condNames
 	itr.conds.curs = conds
 
+	// note: if opt.Condition is nil, itr.m can be nil.
```

Comment:
You can remove this comment. Passing `nil` to the `MapValuer` should be fine regardless. Assigning to it would cause a panic, but that's not relevant to the valuer.

Reply:
Good point, that hadn't occurred to me.